Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8 as base
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+WORKDIR /todo_app
+COPY pyproject.toml poetry.lock /todo_app/
+COPY ./todo_app /todo_app/todo_app
+
+### PRODUCTION ###
+FROM base as production
+RUN poetry install --no-dev
```

Comment:
Nice 👍 

You could also set some other production specific settings, e.g.  `ENV FLASK_ENV=production`